src/cdfmean.f90

```diff
@@ -93,7 +101,8 @@ PROGRAM cdfmean
   narg = iargc()
   IF ( narg == 0 ) THEN
      PRINT *,' usage : cdfmean  IN-file IN-var T|U|V|F|W [imin imax jmin jmax kmin kmax]'
-     PRINT *,'       ... [-full] [-var] [-zeromean] '
+     PRINT *,'       ... [-full] [-var] [-zeromean] [-out_suff OUT-file-suffix]'
+     PRINT *,'       ... [-basins n_basins basin_names..]'
```

Comment:
Do you mean prefix instead of suffix ? If I understand right, it is to add the path and runid (for example) before the ouput file name.